internal/schema/metadataconfig.go

```diff
@@ -0,0 +1,59 @@
+package schema
+
+import (
+	"fmt"
+	"strings"
+
+	"google.golang.org/protobuf/proto"
+
+	schemapb "github.com/TBD54566975/ftl/backend/protos/xyz/block/ftl/v1/schema"
+)
+
+// MetadataConfig represents a metadata block with a list of config items that are used.
+//
+//protobuf:10,optional
+type MetadataConfig struct {
+	Pos Position `parser:"" protobuf:"1,optional"`
+
+	Config []*Ref `parser:"'+' 'config' @@ (',' @@)*" protobuf:"2"`
+}
+
+var _ Metadata = (*MetadataConfig)(nil)
+
+func (m *MetadataConfig) Position() Position { return m.Pos }
+func (m *MetadataConfig) String() string {
+	out := &strings.Builder{}
+	fmt.Fprint(out, "+calls ")
```

Comment:
```suggestion
	fmt.Fprint(out, "+config ")
```

Reply:
Ugh, sorry, this PR is mostly just copypasta from from the calls metadata.